deps/lua/src/ldebug.c

```diff
@@ -94,8 +94,11 @@ LUA_API int lua_getstack (lua_State *L, int level, lua_Debug *ar) {
     ar->i_ci = cast_int(ci - L->base_ci);
   }
   else if (level < 0) {  /* level is of a lost tail call? */
-    status = 1;
-    ar->i_ci = 0;
+    if (ci == L->ci) status = 0; /*   level was negative? */
+    else {/*   level is of a lost tail call */
+      status = 1;
+      ar->i_ci = 0;
+    }
```

Comment:
This is used on the Lua `debug` API (the `debug` global), and so inaccessible on Redis (Redis set `debug='nil'` and so user code can not use it). Still we might want to take this change but it will have no difference on any code execution.